src/functions.jl

```diff
@@ -10,9 +10,62 @@
 Interverts the car `i` with the car `j` in `solution.sequence`. Updates
 `solution.M1`, `solution.M2` and `solution.M3`.
 """
-function move_exchange!(solution::Solution, i::Int, j::Int, instance::Instances)
-    # TODO
-    return Solution
+function move_exchange(pi::Solution,i::Int,j::Int, instance::Instances)
```

Comment:
`pi` is a reserved keyword. Use `solution` instead.

`pi` est un mot réservé. Utilise plutôt `solution`.